experimental/WreathMacdonaldpols/src/WreathMacdonaldpols.jl

```diff
@@ -0,0 +1,256 @@
+# I would like to credit Dario Mathiä who produced an initial version of the following code
+
+import Chevie: complex_reflection_group, charinfo, CharTable, CyclotomicNumbers, Cyc
+export wreath_macs
+
+# Tools
+
+#gives indices of a subarray
+function subarray_indices(sub,arr)
+  res = Int[]
+  for i in 1:length(sub)
+    push!(res,findfirst(x->x==sub[i],arr))
+  end
+  return res
+end
+
+# Computes the b-invariant of a partition
+function b1(lambda::Vector{Int64})
+  if length(lambda)==0
+    return 0
+  end
+  return sum(i->(i-1)*lambda[i],1:length(lambda))
+end
```

Comment:
```suggestion
function b1(lambda::Partition)
  return sum((i-1)*l_i for (i, l_i) in enumerate(lambda); init=0)
end
```
this would a bit more julia-idiomatic

Reply:
Yep, way shorter.